extensions/proposals/EXT_provoking_vertex/extension.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<proposal href="EXT_provoking_vertex/">
+  <name>EXT_provoking_vertex</name>
+  <contact>
+    <a href="https://www.khronos.org/webgl/public-mailing-list/">WebGL working group</a> (public_webgl 'at' khronos.org)
+  </contact>
+  <contributors>
+    <contributor>Kelsey Gilbert (Mozilla)</contributor>
+    <contributor>Members of the WebGL working group</contributor>
+  </contributors>
+  <number>48</number>
+  <depends>
+    <api version="1.0"/>
```

Comment:
Should this depend on 2.0 rather than 1.0, since flat shading was only introduced in ESSL 300?

Reply:
Discussed on today's call - I'll try to edit in place.

Reply:
done!